scripts/internal-benchmark-results.ts

```diff
@@ -0,0 +1,108 @@
+import { BenchmarkRow } from './types';
+
+export const internalBenchmarkResults : BenchmarkRow[] = [
```

Comment:
Which benchmark did you run to get these results?

If I have unknown devices in the future I would be able to run the same benchmark as well.

Reply:
GFXBench Manhattan (the original) through the GFXBench app